src/site/content/en/blog/app-shortcuts/index.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Get things done quickly with app shortcuts
+subhead: App shortcuts give quick access to a handful of common actions that users need frequently.
+authors:
+  - beaufortfrancois
+  - jungkees
+date: 2020-05-27
+updated: 2020-05-27
+hero: hero.jpg
+thumbnail: thumbnail.jpg
+alt: A photo of an Android phone showing an app shortcuts menu
+description: App shortcuts give quick access to a handful of common actions that users need frequently.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - capabilities
+  - fugu
+  - progressive-web-apps
+---
+
+To improve users' productivity and facilitate re-engagement with key tasks, the
+web platform now supports app shortcuts. They allow web developers to provide
+quick access to a handful of common actions that users need frequently.
+
+{% Aside %}
+At the time of writing, app shortcuts are available in Chrome 84 for Android. On
+Windows, you can experiment with them on both Chrome 84 and Edge 84 by enabling
+the flag `about://flags/#enable-desktop-pwas-app-icon-shortcuts-menu`.
+{% endAside %}
+
+This article will teach you how to define those app shortcuts. Additionally,
+you'll learn some associated best practices.
+
+## About app shortcuts
+
+App shortcuts help users quickly start common or recommended tasks within your
+web app. Easy access to those tasks from anywhere the app icon is displayed will
+enhance users' productivity as well as increase their engagement with the web
+app.
+
+The app shortcuts menu is invoked by right-clicking the app icon in the taskbar
+(Windows) or dock (macOS) on the user's desktop, or long pressing the app's
+launcher icon on Android.
+
+<figure class="w-figure">
+  <img src="./app-shortcuts-menu-android.png" alt="Screenshot of an app shortcuts menu opened on Android">
+  <figcaption class="w-figcaption">App shortcuts menu opened on Android</figcaption>
+</figure>
+
+The app shortcuts menu is shown only for [Progressive Web Apps] that are
+installed on the user's desktop or mobile device.
```

Comment:
Would be nice to explicitly define what criteria are relevant here. Does it have to meet the entire base PWA checklist? Or is it just the existence of the app on the homescreen? Specific properties in the manifest?

Reply:
PWA have to be installed to the homescreen, that implies all the stuff covered in https://web.dev/install-criteria/. Shall I link to it this article instead?

Reply:
Say what you just said and link to the article.

Reply:
Don't hold up the article, though. I'd be fine with submitting this change as an update.

Reply:
6b8201d addresses it